nautobot/core/models/mixins.py

```diff
@@ -0,0 +1,14 @@
+class LimitQuerysetChoicesSerializerMixin:
```

Comment:
Is `core/models/mixins.py` the appropriate place for this? Seems like it's specific to the REST API, should it be in `core/api/mixins.py` instead?